docs/website/docs/plus/getting-started/tutorial.md

```diff
@@ -0,0 +1,395 @@
+
+### Create an empty project
+
+Delete all the files in the `tutorial` folder and run the following command to create an empty project:
+
+```sh
+dlt project init
+```
+
+This will create a project without any sources, destinations, datasets, or pipelines; the project will be named after the folder.
+
+### Add all entities
+
+Now we can add all of our entities individually. This way, we can also give them their own names, which will be useful when having multiple destinations of the same type, for example.
+
+Add a source:
+
+```sh
+# add a new arrow source called "my_arrow_source"
+dlt source my_arrow_source add arrow
+```
+
+Add a destination:
+
+```sh
+# add a new duckdb destination called "my_duckdb_destination"
+# this will also create a new dataset called "my_duckdb_destination_dataset"
+dlt destination my_duckdb_destination add duckdb
+```
+
+If you want dlt+ also to add a definition of dataset for this specific destination, you can use the `--dataset-name` flag:
+
+```sh
+# add a new duckdb destination called "my_duckdb_destination"
+dlt destination my_duckdb_destination add duckdb --dataset-name my_duckdb_destination_dataset
+```
+
+In this case both destinations and datasets sections of the `dlt.yml` file will be updated:
+
+```yaml
+# your destinations are the databases where your data will be saved
+destinations:
+  my_duckdb_destination:
+    type: duckdb
+
+# your datasets are the datasets on your destinations where your data will go
+datasets:
+  my_duckdb_destination_dataset:
+    destination:
+    - my_duckdb_destination
+```
+
+Now we can add a pipeline that uses the source and destination we just added:
+
+```sh
+# add a new pipeline called "my_pipeline" which loads from my_arrow_source and saves to my_duckdb_destination
+# we select the my_duckdb_destination_dataset with the optional flag
+dlt pipeline my_pipeline add my_arrow_source my_duckdb_destination --dataset_name my_duckdb_destination_dataset
+```
+
+### Adding the core source
+
+You can add multiple entities using CLI commands. Let's add another source - this time, a core source such as a
+([REST API](../../dlt-ecosystem/verified-sources/rest_api/index.md), [SQL database](../../dlt-ecosystem/verified-sources/sql_database/index.md), [filesystem](../../dlt-ecosystem/verified-sources/filesystem/index.md)).
+
+Run the following command to add an SQL database source named `sql_db_1`:
+```sh
+# add a new sql_database source called "sql_db_1"
+dlt source sql_db_1 add sql_database
+```
+
+This will add the new source to your `dlt.yml` file:
+
+```yaml
+sources:
+  arrow:
+    type: sources.arrow.source
+
+  sql_db_1:
+    type: sql_database
+```
+
+The corresponding credential placeholders will be added to `.dlt/secrets.toml`, but you can also define them in `dlt.yml`.
```

Comment:
I think you can only define configs and not secrets in the yaml, we should explain this better somewhere at some point..

Reply:
I don't think it's true
Works for me
<img width="613" alt="Screenshot 2025-02-18 at 11 39 09" src="https://github.com/user-attachments/assets/87cb3229-2de4-49ad-abf7-7b10a6cca5a1" />